lib/lighthouse/facilities/client.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+require 'common/client/base'
+require 'lighthouse/facilities/response'
+
+module Lighthouse
+  module Facilities
+    # Documentation located at:
+    # https://developer.va.gov/explore/facilities/docs/facilities
+    class Client < Common::Client::Base
+      attr_accessor :headers
+
+      configuration Lighthouse::Facilities::Configuration
+
+      def initialize(api_key)
+        self.headers = { 'apikey' => api_key }
+      end
+
+      def get_by_id(id)
+        response = perform(:get, "/va_facilities/v0/facilities/#{id}", nil, headers)
+        Lighthouse::Facilities::Response.new(response.body, response.status).new_facility
+      end
+
+      def get_facilities(params)
```

Comment:
A usage example of what params are available would be nice for usability here!

Reply:
Added method signatures